AssetStoreBatchMode/AssetStoreBatchMode.cs

```diff
@@ -325,7 +328,10 @@ static string GetDraftAssetsPath(string localRootPath)
     }
     static void Export(Package package, string localRootPath, string toPath)
     {
-        File.Delete(toPath);
+        if (File.Exists(toPath))
```

Comment:
Not sure this check is actually needed - from File.Delete docs

> If the file to be deleted does not exist, no exception is thrown.

Were there problems here for you?

Reply:
I thought I had a problem with that piece of code. I just tried to reproduce it without my change but I couldn't. Maybe I had some file handle open on it. You are right, it does not throw an exception, my bad. I removed my change.
By the way, thanks for this repo! ;) Really helped me to automate the asset store package upload.